src/main/java/Duke.java

```diff
@@ -1,10 +1,203 @@
+            boolean isDeadline = words[0].equals("deadline");
+            boolean isEvent = words[0].equals("event");
+            boolean isDelete = words[0].equals("delete");
+            try {
+                validateInput(words);
+            } catch (Exception e){
+                System.out.println(e);
+                line = Input.nextLine();
+                inSystem = !line.equals("bye");
+                continue;
+            }
+            if(isList){
+                try {
+                    validateListInput(words);
+                } catch (Exception e) {
+                    System.out.println(e);
+                    line = Input.nextLine();
+                    inSystem = !line.equals("bye");
+                    continue;
+                }
+                printDashLine();
+                System.out.println("Here are the tasks in your list:");
+                printList(Tasks);
+                printDashLine();
+            }
+            else if(isDone) {
+                int taskNumber = Integer.parseInt(words[1]) - 1;
+                Tasks.get(taskNumber).taskComplete();
+                printDashLine();
+                System.out.println("Nice! I've marked this task as done:\n" + " " + Tasks.get(taskNumber).toString());
+                printDashLine();
+            }
+            else if(isDelete) {
+                int taskNumber = Integer.parseInt(words[1]) - 1;
+                printDashLine();
+                System.out.println("Noted. I've removed this task:\n" + Tasks.get(taskNumber).toString());
+                Tasks.remove(taskNumber);
+                System.out.println("Now you have " + Tasks.size() + " tasks in the list.");
+                printDashLine();
+            }
+            else if (isTodo){
+                try {
+                    validateToDoInput(words);
+                } catch (Exception e) {
+                    System.out.println(e);
+                    line = Input.nextLine();
+                    inSystem = !line.equals("bye");
+                    continue;
+                }
+                line = line.replace("todo ", "");
+                ToDo toDo = new ToDo(line);
+                Tasks.add(toDo);
+                printTaskAdded(Tasks);
+            }
+            else if (isDeadline){
+                line = line.replace("deadline ", "");
+                words = line.split("/by ");
+                Deadline deadline = new Deadline(words[0], words[1]);
+                Tasks.add(deadline);
+                printTaskAdded(Tasks);
+            }
+            else if (isEvent) {
+                line = line.replace("event ", "");
+                words = line.split("/at ");
+                Event event = new Event(words[0], words[1]);
+                Tasks.add(event);
+                printTaskAdded(Tasks);
+            }
+            line = Input.nextLine();
+            inSystem = !line.equals("bye");
+        }
+        String file = "tasks.txt";
```

Comment:
`fileName` and store it somwehere else, maybe a constant above will do since it does not get altered here